src/main/java/gov/cabinetoffice/gap/applybackend/web/GrantMandatoryQuestionsController.java

```diff
@@ -0,0 +1,101 @@
+    }
+
+    @GetMapping("/{mandatoryQuestionId}")
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "200", description = "Grant Mandatory found", content = @Content(mediaType = "application/json", schema = @Schema(implementation = GrantMandatoryQuestions.class))),
+            @ApiResponse(responseCode = "403", description = "User cannot access this mandatory question", content = @Content(mediaType = "application/json")),
+            @ApiResponse(responseCode = "404", description = "No Grant Mandatory question found", content = @Content(mediaType = "application/json")),
+    })
+    public ResponseEntity<GetGrantMandatoryQuestionDto> getGrantMandatoryQuestionsById(@PathVariable final UUID mandatoryQuestionId) {
+        final JwtPayload jwtPayload = (JwtPayload) SecurityContextHolder.getContext().getAuthentication().getPrincipal();
+        final GrantMandatoryQuestions grantMandatoryQuestions = grantMandatoryQuestionService.getGrantMandatoryQuestionById(mandatoryQuestionId, jwtPayload.getSub());
+        final GetGrantMandatoryQuestionDto getGrantMandatoryQuestionDto = modelMapper.map(grantMandatoryQuestions, GetGrantMandatoryQuestionDto.class);
+
+        return ResponseEntity.ok(getGrantMandatoryQuestionDto);
+    }
+
+    @PatchMapping("/{mandatoryQuestionId}")
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "200", description = "Grant Mandatory question updated", content = @Content(mediaType = "application/json", schema = @Schema(implementation = String.class))),
+            @ApiResponse(responseCode = "403", description = "User cannot access this mandatory question", content = @Content(mediaType = "application/json")),
+            @ApiResponse(responseCode = "404", description = "No Grant Mandatory question found", content = @Content(mediaType = "application/json")),
+    })
+    public ResponseEntity<String> updateMandatoryQuestion(@PathVariable final UUID mandatoryQuestionId,
+                                                          @RequestBody @Valid final UpdateGrantMandatoryQuestionDto mandatoryQuestionDto) {
+        final JwtPayload jwtPayload = (JwtPayload) SecurityContextHolder.getContext().getAuthentication().getPrincipal();
+        final GrantMandatoryQuestions grantMandatoryQuestions = grantMandatoryQuestionService.getGrantMandatoryQuestionById(mandatoryQuestionId, jwtPayload.getSub());
+
+        modelMapper.map(mandatoryQuestionDto, grantMandatoryQuestions);
+        if (mandatoryQuestionDto.getOrgType() != null) {
+            grantMandatoryQuestions.setOrgType(GrantMandatoryQuestionOrgType.valueOfName(mandatoryQuestionDto.getOrgType()));
+        }
+        if (mandatoryQuestionDto.getFundingAmount() != null) {
+            grantMandatoryQuestions.setFundingAmount(new BigDecimal(mandatoryQuestionDto.getFundingAmount()));
+        }
+        grantMandatoryQuestions.setId(mandatoryQuestionId);
+        grantMandatoryQuestions.setStatus(GrantMandatoryQuestionStatus.IN_PROGRESS);
```

Comment:
I don't think we want to set it to In Progress any time we update.
The user can complete their mandatory questions, and change the answers later and that shouldn't set the status back to in progress.
There will be more complicated logic added here later so its fine to put a TODO for now.

Reply:
added an if to change the status only if it was NOT_STARTED